pagefaultgames/pokerogue
# [Bug] Fix PostSummonPhase Logic to Support Single Pokémon Teams

## What are the changes?
I've fixed the condition for `postSummonPhase` that checks how many Pokémon there were on the field for double battles. It was lacking a condition for when there is only 1 Pokémon available in our team, even if it's a double battle.


## What did change?
- **Before:** The `postSummonPhase` condition only checked for the number of Pokémon on the field for double battles, without considering the scenario of having only 1 Pokémon available in the team.
- **After:** Added a condition to handle the case where there is only 1 Pokémon available in our team during a double battle.

### Screenshots/Videos
![webstorm64_qlcdP2nrjQ](https://github.com/pagefaultgames/pokerogue/assets/44787002/21d49723-5dbc-4e2a-9a57-075a918a056c)


## How to test the changes?
1. Check out the branch with the changes.
2. Run the application and navigate to a double battle scenario.
3. Ensure that there is only 1 Pokémon available in the team.
4. Observe the `postSummonPhase` behavior and verify that it now correctly handles the scenario.
5. Run all unit tests to ensure no other parts of the code are affected.
    - `npm run test`

## Checklist
